src/lib/rng/system_rng/system_rng.cpp

```diff
@@ -211,7 +212,7 @@ class System_RNG_Impl final : public RandomNumberGenerator {
          uint8_t* buf = output.data();
          size_t len = output.size();
          while(len > 0) {
-            const ssize_t got = ::getrandom(buf, len, flags);
```

Comment:
You accidentally removed the variable name `got`, re-adding it should fix the CI.